stable/traefik/values.yaml

```diff
@@ -19,6 +19,7 @@ cpuLimit: 100m
 memoryLimit: 30Mi
 debug:
   enabled: false
+deploymentStrategy: {}
```

Comment:
Rather than force people to go explore documentation, let's put a sample configuration for this here (instead of empty object `{}`) and leave the whole thing commented.

Reply:
@krancour Other configuration keys use empty objects as well (`nodeSelector`, `affinity`). So that should be fine, right? I added example documentation however, so users will more easily see what the configuration key is used for.